Vagrantfile

```diff
@@ -1,6 +1,10 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "fedora/30-cloud-base"
-  config.vm.provision "shell", path:"ProvisionScripts.sh"
-end
+  config.vm.provider "virtualbox" do |v|
+    v.name = "tiger_dev"
+    v.memory = 2048
```

Comment:
why did you increased the memory? is it required?

Reply:
Thought we meant for it to be so because of the presentation. fixed.